code/style.css

```diff
@@ -1,4 +1,220 @@
-html {
-  background: #0025ff;
+body {
+  background-color: #f5f5f5;
+  font-family: "Open Sans",sans-serif;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+/** HEADER **/
+
+.header {
+  background-color: rgba(95, 158, 160, 0.842);
+  height: 100px;
+  display: flex;
+  align-items: center;
+  padding: 0 10px;
+  border: 10px solid white;
+  box-shadow: 1px 3px 5px rgba(117, 157, 158, 0.2);
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  width: 30px;
+  height: 30px;
+  justify-content: space-around;
+  margin-right: 10px;
+}
+
+.hamburger div {
+  background-color: #fff;
+  width: 30px;
+  height: 5px;
+  border-radius: 2px;
+}
+
+.top-heading {
+  color: #fff;
+  font-size: 30px;
+  font-family: 'Lobster', cursive;
+}
+
+.top-heading:hover {
+  color: pink;
+}
+
+.logo {
+  width: 50px;
+  order: 1;
+  margin-left: auto;
+}
+
+.navigation-container {
+  display: none;
+}
+
+.navigation-item {
+  margin-right: 20px;
   color: #fff;
+  text-decoration: none;
+}
+
+.navigation-item:hover {
+  color: pink;
+}
+
+.container {
+  width: 330px;
+  margin: 30px auto;
```

Comment:
We can use flexbox to center! Try this:
```
body {
  display: flex;
  flex-direction: column;
  align-items: center;
}
```

Reply:
Also, it would be nice to use the whole width, there's a lot of whitespace:

<img width="656" alt="Skärmavbild 2021-02-02 kl  13 37 45" src="https://user-images.githubusercontent.com/51529201/106601351-d7177480-655b-11eb-8166-2ccf3babb808.png">